spec/spec_helper.rb

```diff
@@ -0,0 +1,22 @@
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
+$LOAD_PATH.unshift(File.dirname(__FILE__))
+require 'rspec'
+require 'bourbon'
+require 'aruba/api'
+require 'css_parser'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.